### PR TITLE
chore(speech): migrate code from googleapis/python-speech

### PR DESCRIPTION
**Pull request by telpirion**
- Add generated code samples. (#9153)
- Update samples manifest (via synth). (#9211)
- codegen(speech): reorder samples (#9313)
- chore(speech): bump copyright year to 2020, tweak docstring formatting (via synth) (#10238)
- speech: remove erroneous comment (#17)
- Add quickstart sample for the Cloud Speech API. [(#658)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/658)
- Update readmes
- Auto-update dependencies. [(#715)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/715)
- Add cloud-client samples for speech [(#775)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/775)
- Adds usage examples to api-client samples and fixes copy/paste error in cloud-client/transcribe. [(#782)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/782)
- Add speech GCS samples [(#784)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/784)
- Updates client version for compatibility with language [(#842)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/842)
- Adds streaming snippet for howto [(#881)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/881)
- Remove resource [(#890)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/890)
- Speech v1 [(#889)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/889)
- Changes docs for async playback to point to LINEAR16 example [(#897)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/897)
- Speech version bump [(#899)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/899)
- Update transcribe_async.py to have long GCS flac example [(#904)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/904)
- Re-generate all readmes
- Fix typo [(#926)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/926)
- Fix README rst links [(#962)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/962)
- Auto-update dependencies. [(#1004)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1004)
- Speech sample using microphone [(#1013)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1013)
- Speech gapic client library [(#1012)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1012)
- use region tags [(#1022)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1022)
- Auto-update dependencies. [(#1039)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1039)
- add word time offsets to async sample [(#1042)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1042)
- Add word time offset samples [(#1050)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1050)
- Remove old parameter enable_word_time_offsets [(#1054)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1054)
- Auto-update dependencies. [(#1055)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1055)
- Auto-update dependencies. [(#1093)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1093)
- Print all results, not all alternatives [(#1098)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1098)
- quickstart: print out all results. [(#1108)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1108)
- Update all generated readme auth instructions [(#1121)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1121)
- Added Link to Python Setup Guide [(#1158)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1158)
- Auto-update dependencies. [(#1186)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1186)
- Added "Open in Cloud Shell" buttons to README files [(#1254)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1254)
- Speech: clarify comments. [(#1278)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1278)
- Auto-update dependencies. [(#1320)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1320)
- Auto-update dependencies. [(#1355)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1355)
- Auto-update dependencies. [(#1359)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1359)
- Speech model selection [(#1361)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1361)
- Auto-update dependencies. [(#1377)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1377)
- Speech API: enhanced model and recognition metadata [(#1436)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1436)
- Regenerate the README files and fix the Open in Cloud Shell link for some samples [(#1441)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1441)
- add Speech API auto punctuation sample [(#1446)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1446)
- use unicode [(#1453)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1453)
- Update READMEs to fix numbering and add git clone [(#1464)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1464)
- Diarization [(#1556)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1556)
- Multi Channel [(#1561)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1561)
-  Multi Langauge [(#1564)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1564)
- Added the sample for Word Level Confidence [(#1567)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1567)
- Diarization Output Modified [(#1586)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1586)
- Speech region tag update [(#1644)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1644)
- Update enhanced models and auto punctuation to GA [(#1702)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1702)
- Test form of result only, not contents [(#1866)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1866)
- speech: updated README for model selection sample [(#1892)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1892)
- Speech continuous [(#1940)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1940)
- Add missing import for code sample [(#1898)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1898)
- Auto-update dependencies. [(#1980)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1980)
- add speech api multichannel samples [(#2003)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2003)
- Move mic samples out of cloud-client so that cloud-client samples can be run in cloud shell [(#2062)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2062)
- Move mic samples out of cloud-client so that cloud-client samples can be run in cloud shell [(#2062)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2062)
- Place import inside the function [(#2209)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2209)
- adding infinite streaming sample to samples folder, unlike indefinite… [(#2161)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2161)
- Increase timeout to 5 mins [(#2206)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2206)
- Python 2 compatibility for print statements [(#2226)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2226)
- Enhanced models are available to all projects [(#2207)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2207)
- Adds updates for samples profiler ... vision [(#2439)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2439)
- Adds updates for samples profiler ... vision [(#2439)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2439)
- Update transcribe_streaming.py [(#2597)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2597)
- Auto-update dependencies. [(#2005)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2005)
- speech: fixing flaky auto punctuation test [(#2747)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2747)
- speech: canonical sample for speechcontext ga of classes [(#2756)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2756)
- Speech sample cleanup [(#2628)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2628)
- chore(deps): update dependency google-cloud-speech to v1.3.2 [(#3167)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3167)
- chore(deps): update dependency google-cloud-speech to v1.3.2 [(#3167)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3167)
- chore(deps): update dependency six to v1.14.0 [(#3200)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3200)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- chore: some lint fixes [(#3750)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3750)
- chore(deps): update dependency six to v1.15.0 [(#3853)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3853)
- chore(deps): update dependency pytest to v5.4.3 [(#4279)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4279)
- chore(deps): update dependency pytest to v5.4.3 [(#4279)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4279)
- chore(deps): update dependency mock to v4 [(#4287)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4287)
- feat(speech): Add Speech-to-Text On-Prem sample [(#4223)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4223)
- Update dependency pytest to v6 [(#4390)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4390)
- Update dependency pytest to v6 [(#4390)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4390)
- chore: update templates
- chore: delete duplicate generated samples
- chore: fix sample test
- chore: fix lint
- feat!: migrate to microgenerator (#61)
- fix: migrated samples to speech 2.0.0 (#78)
- fix: deleted a line duplicating the call to the recognizer  (#83)
- chore(deps): update dependency google-cloud-speech to v2.0.1 (#88)
- docs: updated setup documentation to point to python-speech instead of python-docs-samples (#89)
- chore: updated quickstart (#94)
- fix: Remove incorrect comment on enhanced models (#95)
- feat: adds new multi region sample (#96)
- chore: update templates (#101)
- feat(v1p1beta1): support Model Adaptation (#104)
- chore(deps): update dependency google-cloud-speech to v2.1.0 (#116)
- feat: adds model adaptation sample (#121)
- chore: update templates (#127)
- chore(deps): update dependency google-cloud-speech to v2.2.0 (#133)
- chore(deps): update dependency google-cloud-speech to v2.2.1 (#139)
- chore(deps): update dependency google-cloud-speech to v2.3.0 (#146)
- chore(deps): update dependency six to v1.16.0 (#154)
- chore: migrate to owl bot (#158)
- chore: changed assert output to check constant values (#161)
- chore: new owl bot post processor docker image (#165)
- chore(deps): update dependency pytest to v6.2.4 (#160)
- chore(deps): update dependency google-cloud-speech to v2.4.0 (#168)
- chore(deps): update dependency mock to v4.0.3 (#159)
- chore: Fix typo in sample code (#112)
- chore: fix region problem (#171)
- Update speech_model_adaptation_beta.py
- Update speech_model_adaptation_beta_test.py
- samples: adds speech export to gcs sample (#176)
- chore: new owl bot post processor docker image (#177)
- chore(deps): update dependency google-cloud-speech to v2.4.1 (#187)
- chore(deps): update dependency google-cloud-storage to v1.39.0 (#185)
- chore(deps): update dependency google-cloud-storage to v1.40.0 (#192)
- chore(deps): update dependency google-cloud-speech to v2.5.0 (#194)
- chore(deps): update dependency google-cloud-storage to v1.41.0 (#206)
- chore(deps): update dependency google-cloud-storage to v1.41.1 (#213)
- feat: add Samples section to CONTRIBUTING.rst (#211)
- chore(deps): update dependency google-cloud-speech to v2.5.1 (#214)
- docs(samples): Add Spoken Punctuation and Emojis code samples (#155)
- chore(deps): update dependency google-cloud-speech to v2.6.0 (#220)
- chore(deps): update dependency google-cloud-speech to v2.7.0 (#226)
- docs(samples): remove io dependency in transcribe samples (#223)
- chore(deps): update dependency google-cloud-storage to v1.42.0 (#230)
- chore: remove transcribe_async.py (redundant) as last step in refactor. (#231)
- chore: drop mention of Python 2.7 from templates (#233)
- chore(deps): update dependency google-cloud-speech to v2.8.0 (#243)
- chore(deps): update dependency pytest to v6.2.5 (#240)
- docs(samples): fixed model adaptation sample to correctly refer to custom classes (#246)
- chore(deps): update dependency google-cloud-storage to v1.42.1 (#249)
- chore: blacken samples noxfile template (#250)
- chore(deps): update all dependencies (#251)
- chore(deps): update dependency google-cloud-speech to v2.9.1 (#258)
- chore: fail samples nox session if python version is missing (#261)
- chore(deps): update dependency google-cloud-storage to v1.42.3 (#263)
- fix: Remove reference to deprecated message "diarization_speaker_count" in favor of diarization_config (#264)
- chore(deps): update dependency google-cloud-speech to v2.9.3 (#266)
- chore(python): Add kokoro configs for python 3.10 samples testing (#271)
- chore(deps): update dependency google-cloud-speech to v2.10.0 (#273)
- chore(deps): update dependency google-cloud-speech to v2.11.0 (#278)
- chore(deps): update dependency google-cloud-speech to v2.11.1 (#281)
- docs(samples): a draft that refactors speech_to_storage_beta sample (#276)
- chore(deps): update dependency google-cloud-storage to v1.43.0 (#288)
- chore: update python-docs-samples link to main branch (#293)
- chore(samples): Add check for tests in directory (#301)
- chore(deps): update dependency google-cloud-storage to v1.44.0 (#298)
- chore(deps): update dependency google-cloud-storage to v2 (#305)
- chore(deps): update dependency google-cloud-speech to v2.12.0 (#307)
- chore(python): Noxfile recognizes that tests can live in a folder (#308)
- docs: add generated snippets (#321)
- chore: use gapic-generator-python 0.63.4 (#323)
- chore: update copyright year to 2022 (#325)
- chore(deps): update dependency google-cloud-storage to v2.1.0 (#309)
- chore(deps): update all dependencies (#326)
- chore(deps): update all dependencies (#327)
- chore: Adding support for pytest-xdist and pytest-parallel (#335)
- chore(deps): update dependency google-cloud-speech to v2.13.1 (#338)
- chore(deps): update dependency pytest to v7.1.0 (#340)
- chore(deps): update dependency google-cloud-storage to v2.2.0 (#341)
- chore(deps): update dependency google-cloud-storage to v2.2.1 (#342)
- chore(deps): update dependency pytest to v7.1.1 (#343)
- chore(python): use black==22.3.0 (#346)
- chore(deps): update dependency google-cloud-storage to v2.3.0 (#360)
- chore: use gapic-generator-python 0.65.1 (#361)
- chore(python): add nox session to sort python imports (#362)
- chore(deps): update dependency pytest to v7.1.2 (#365)
- feat: Add adaptation proto for v1 api (#371)
- chore(deps): update dependency google-cloud-speech to v2.14.0 (#374)
- fix: library deprecation warnings (#376)
- fix: require python 3.7+ (#414)
- chore(deps): update all dependencies (#404)
- chore(deps): update all dependencies (#421)
- chore(deps): update all dependencies (#423)
- chore(deps): update dependency google-cloud-speech to v2.15.1 (#428)
- chore(deps): update dependency pytest to v7.1.3 (#438)
- chore: Bump gapic-generator-python version to 1.3.0 (#439)
- chore: detect samples tests in nested directories (#442)
- feat: added Speech-to-Text V2 API (#447)
- chore(deps): update dependency google-cloud-speech to v2.16.0 (#449)
- chore(deps): update dependency google-cloud-speech to v2.16.1 (#454)
- docs(samples): add create_recognizer code sample (#450)
- docs(samples): add quickstart code sample for STT V2 (#451)
- samples: Code samples for Speech-to-Text V2 (#460)
- chore(deps): update dependency google-cloud-speech to v2.16.2 (#459)
- docs(samples): Update code samples for adaptation and VAD (#462)
- chore(deps): update dependency pytest to v7.2.0 (#464)
- chore(deps): update dependency google-cloud-storage to v2.6.0 (#467)
- chore(python): drop flake8-import-order in samples noxfile (#470)
- chore(deps): update dependency google-cloud-storage to v2.7.0 (#471)
- feat: Added new fields to facilitate debugging (#465)
- chore(deps): update all dependencies (#473)
- chore(python): add support for python 3.11 (#475)
- chore(deps): update dependency mock to v5.0.1 (#477)

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by kweinmeister** [comment]
2.7 build error:
```
#
# speech/snippets
#

------------------------------------------------------------
- testing /workspace/speech/snippets
------------------------------------------------------------
nox > Error while collecting sessions.
nox > Sessions not found: py-2.7
No user noxfile_config found: detail: No module named 'noxfile_config'
PWD: /workspace/speech/snippets

 Testing failed: Nox returned a non-zero exit code. 
```

**Review by nicain — APPROVED** [review]

**Comment by kweinmeister** [comment]
@telpirion Should we include the CODEOWNERS and blunderbuss.yml updates, too?

@nicain @m-strzelczyk @dandhlee These changes are blocked by an invalid license header, "Google Inc" rather than "Google, Inc." Given that these are existing files, should we update the configuration to allow "Google Inc" or do you have another suggestion to move forward?

**Comment by dandhlee** [comment]
We should fix the license headers, if they're deemed invalid (slight tweak to fix them is fine).

File: speech/microphone/noxfile_config.py
Comment (dandhlee) [inline speech/microphone/noxfile_config.py:1]: Remove this file.

File: speech/snippets/noxfile_config.py
Comment (dandhlee) [inline speech/snippets/noxfile_config.py:1]: Ditto.

File: speech/microphone/transcribe_streaming_mic.py
Comment (dandhlee) [inline speech/microphone/transcribe_streaming_mic.py:3]:
```suggestion
# Copyright 2017 Google LLC
```

File: speech/microphone/transcribe_streaming_mic_test.py
Comment (dandhlee) [inline speech/microphone/transcribe_streaming_mic_test.py:1]:
```suggestion
# Copyright 2017 Google LLC

```

File: speech/snippets/adaptation_v2_custom_class_reference.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_custom_class_reference.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/adaptation_v2_custom_class_reference_test.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_custom_class_reference_test.py:1]:
```suggestion
# Copyright 2017 Google LLC

```

File: speech/snippets/adaptation_v2_inline_custom_class.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_inline_custom_class.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/adaptation_v2_inline_custom_class_test.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_inline_custom_class_test.py:1]:
```suggestion
# Copyright 2022 Google LLC

```

File: speech/snippets/adaptation_v2_inline_phrase_set.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_inline_phrase_set.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/adaptation_v2_inline_phrase_set_test.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_inline_phrase_set_test.py:1]:
```suggestion
# Copyright 2022 Google LLC

```

File: speech/snippets/adaptation_v2_phrase_set_reference.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_phrase_set_reference.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/adaptation_v2_phrase_set_reference_test.py
Comment (dandhlee) [inline speech/snippets/adaptation_v2_phrase_set_reference_test.py:1]:
```suggestion
# Copyright 2022 Google LLC

```

File: speech/snippets/beta_snippets.py
Comment (dandhlee) [inline speech/snippets/beta_snippets.py:3]:
```suggestion
# Copyright 2018 Google LLC
```

File: speech/snippets/beta_snippets_test.py
Comment (dandhlee) [inline speech/snippets/beta_snippets_test.py:1]:
```suggestion
# Copyright 2018 Google LLC

```

File: speech/snippets/create_recognizer.py
Comment (dandhlee) [inline speech/snippets/create_recognizer.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/create_recognizer_test.py
Comment (dandhlee) [inline speech/snippets/create_recognizer_test.py:1]:
```suggestion
# Copyright 2022 Google LLC

```

File: speech/snippets/quickstart.py
Comment (dandhlee) [inline speech/snippets/quickstart.py:3]:
```suggestion
# Copyright 2016 Google LLC
```

File: speech/snippets/quickstart_test.py
Comment (dandhlee) [inline speech/snippets/quickstart_test.py:1]:
```suggestion
# Copyright 2016 Google LLC
```

File: speech/snippets/quickstart_v2.py
Comment (dandhlee) [inline speech/snippets/quickstart_v2.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/quickstart_v2_test.py
Comment (dandhlee) [inline speech/snippets/quickstart_v2_test.py:1]:
```suggestion
# Copyright 2022 Google LLC

```

File: speech/snippets/transcribe.py
Comment (dandhlee) [inline speech/snippets/transcribe.py:3]:
```suggestion
# Copyright 2017 Google LLC
```

File: speech/snippets/transcribe_async_file.py
Comment (dandhlee) [inline speech/snippets/transcribe_async_file.py:1]:
```suggestion
# Copyright 2021 Google LLC
```

File: speech/snippets/transcribe_async_file_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_async_file_test.py:1]:
```suggestion
# Copyright 2021 Google LLC

```

File: speech/snippets/transcribe_async_gcs.py
Comment (dandhlee) [inline speech/snippets/transcribe_async_gcs.py:1]:
```suggestion
# Copyright 2021 Google LLC
```

File: speech/snippets/transcribe_async_gcs_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_async_gcs_test.py:1]:
```suggestion
# Copyright 2021 Google LLC

```

File: speech/snippets/transcribe_auto_punctuation_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_auto_punctuation_test.py:1]:
```suggestion
# Copyright 2018 Google LLC

```

File: speech/snippets/transcribe_context_classes_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_context_classes_test.py:1]:
```suggestion
# Copyright 2020 Google LLC

```

File: speech/snippets/transcribe_enhanced_model_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_enhanced_model_test.py:1]:
```suggestion
# Copyright 2018 Google LLC

```

File: speech/snippets/transcribe_file_v2.py
Comment (dandhlee) [inline speech/snippets/transcribe_file_v2.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/transcribe_file_v2_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_file_v2_test.py:1]:
```suggestion
# Copyright 2022 Google LLC

```

File: speech/snippets/transcribe_gcs_v2.py
Comment (dandhlee) [inline speech/snippets/transcribe_gcs_v2.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/transcribe_gcs_v2_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_gcs_v2_test.py:1]:
```suggestion
# Copyright 2022 Google LLC

```

File: speech/snippets/transcribe_model_selection.py
Comment (dandhlee) [inline speech/snippets/transcribe_model_selection.py:3]:
```suggestion
# Copyright 2017 Google LLC
```

File: speech/snippets/transcribe_model_selection_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_model_selection_test.py:1]:
```suggestion
# Copyright 2016 Google LLC

```

File: speech/snippets/transcribe_multichannel.py
Comment (dandhlee) [inline speech/snippets/transcribe_multichannel.py:3]:
```suggestion
# Copyright 2019 Google LLC
```

File: speech/snippets/transcribe_onprem/transcribe_onprem.py
Comment (dandhlee) [inline speech/snippets/transcribe_onprem/transcribe_onprem.py:3]:
```suggestion
# Copyright 2020 Google LLC

```

File: speech/snippets/transcribe_streaming.py
Comment (dandhlee) [inline speech/snippets/transcribe_streaming.py:3]:
```suggestion
# Copyright 2017 Google LLC
```

File: speech/snippets/transcribe_streaming_test.py
Comment (dandhlee) [inline speech/snippets/transcribe_streaming_test.py:1]:
```suggestion
# Copyright 2017 Google LLC

```

File: speech/snippets/transcribe_streaming_v2.py
Comment (dandhlee) [inline speech/snippets/transcribe_streaming_v2.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

File: speech/snippets/transcribe_streaming_voice_activity_events.py
Comment (dandhlee) [inline speech/snippets/transcribe_streaming_voice_activity_events.py:1]:
```suggestion
# Copyright 2022 Google LLC
```

**Review by dandhlee — CHANGES_REQUESTED** [review]
Please add `CODEOWNERS` and `blunderbuss.yml` for `api: speech`.

File: speech/snippets/noxfile_config.py
Comment (telpirion) [reply speech/snippets/noxfile_config.py]: Done.

File: speech/snippets/transcribe_streaming_voice_activity_timeouts.py
Comment (telpirion) [reply speech/snippets/transcribe_streaming_voice_activity_timeouts.py]: Done.

File: speech/snippets/transcribe_streaming_voice_activity_timeouts_test.py
Comment (telpirion) [reply speech/snippets/transcribe_streaming_voice_activity_timeouts_test.py]: Done.

File: speech/snippets/transcribe_word_time_offsets_test.py
Comment (telpirion) [reply speech/snippets/transcribe_word_time_offsets_test.py]: Done.

**Comment by telpirion** [comment]
Thank you, @dandhlee !

**Comment by dandhlee** [comment]
Seems like tests are passing now! I don't anticipate 3.9 failing for a different reason. Overriding CLA and admin merging.

**Pull request merged**
